modules/page-memory/src/main/java/org/apache/ignite/internal/pagememory/persistence/PersistentPageMemory.java

```diff
@@ -275,6 +283,15 @@ public PersistentPageMemory(
         this.writeThrottle = null;
     }
 
+    private void initMetrics() {
+        metricSource.addMetric(new IntGauge(
+                "CheckpointBufferPagesCount", "Number of pages used in checkpoint buffer.", this::usedCheckpointBufferPages
+        ));
+        metricSource.addMetric(new IntGauge(
+                "CheckpointBufferPagesSize", "Number of used pages in checkpoint buffer.", this::maxCheckpointBufferPages
```

Comment:
Maybe my English is not very good, but I did not really understand how this description is fundamentally different from the description above. If everything is ok, then close my comment.

Reply:
Honestly, I did not check these descriptions. Seems like they say the same thing, I'll improve them

Reply:
Technically, there is a difference between these statements, but I agree that we can re-phrase it